django/db/migrations/operations/models.py

```diff
@@ -802,6 +805,7 @@ def state_forwards(self, app_label, state):
         model_state = state.models[app_label, self.model_name_lower]
         indexes = model_state.options[self.option_name]
         model_state.options[self.option_name] = [idx for idx in indexes if idx.name != self.name]
+        state.reload_model(app_label, self.model_name_lower, delay=True)
```

Comment:
True, but they test the behavior of the respective operations.

It's true that `test_state.py` has a few tests that check for the correct `state_forwards()` behavior, though those tests are focused on the bigger picture of how `ProjectState` and `ModelState` work. Whereas `test_operations.py` tests for the particular implementations of the different migration operations, and as such should hold (regression) tests that focus on a particular implementation of such an migration operation. Thus, `test_operations.py` sounds like the right choice to me.